docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,106 @@
+---
+title: NuGet 5.5 Release Notes
+description: Release notes for NuGet 5.5 including new features, bug fixes, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 03/19/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.5 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.5.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.5](https://visualstudio.microsoft.com/downloads/) | [3.1.200](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.5
+
+* Several accessibility fixes in PM UI
+
+* dotnet.exe reliability fixes with cross platform authentication plugins
```

Comment:
> dotnet.exe reliability fixes with cross platform authentication plugins [](start = 1, length = 72)

Are these fixes significant enough to warrant being mentioned in this section?

Reply:
There were 5-6 issues related to it. 

https://github.com/NuGet/Home/issues/7842
https://github.com/NuGet/Home/issues/8528
https://github.com/NuGet/Home/issues/7699
https://github.com/NuGet/Home/issues/8528

In that list there are issues with >50, >20 comments

Reply:
@nkolev92 - In the next commit, I will add these issues as nested items under `* dotnet.exe reliability fixes with cross platform authentication plugins` feature.